src/styles/styledComponents/Typography.tsx

```diff
@@ -112,6 +112,10 @@ export const StyledBodyMedium = styled(Typography)`
   letter-spacing: ${variables.letterSpacing.lg};
 `;
 
+export const StyledBodyMediumMedium = styled(StyledBodyMedium)`
```

Comment:
Please, remove. In design there is a mistake (Andrei will fix it). For Atkinson font there is only two types of font-weight: 400 and 700. You saw an old design with noto-sans font and old font-weight.